src/scripts/controllers/editClientAddressController.ts

```diff
@@ -1,82 +1,45 @@
 import type { Request, Response, NextFunction } from 'express';
-import { validationResult, type Result } from 'express-validator';
-import { safeString, hasProperty } from '#src/scripts/helpers/index.js';
-import { formatValidationError, handleValidationErrors, type ValidationErrorData } from '#src/scripts/helpers/ValidationErrorHelpers.js';
-import { apiService } from '#src/services/apiService.js';
+import 'csrf-sync'; // Import to ensure CSRF types are loaded
+import { handleGetEditForm, handlePostEditForm, extractFormFields } from '#src/scripts/helpers/index.js';
```

Comment:
Question - I don't get why `src/scripts/controllers/editClientAddressController.ts` exists as a separate controller to `src/scripts/controllers/editClientDetailsController.ts`?

Reply:
As part of the [EL-2614 ](https://dsdmoj.atlassian.net/browse/EL-2614)we will separate out name, phoneNumber and email to its own controller.